highfive/newpr.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
Is this going to cause problems if the original shebang was the correct one?

Reply:
As long as /usr/bin/python is the first python on the PATH of the environment you are concerned about, it shouldn't change anything. If `which python` is saying /usr/bin/python in that environment, then you should be good.

This is a typical way that Python scripts provide flexibility for interpreters being in different places. I wouldn't have noticed if my /usr/bin/python wasn't so old.

Reply:
Cool, thanks. I guess the only problem is if Python isn't in the PATH and then you could just add it.